.gitignore

```diff
@@ -23,3 +23,5 @@ dogpile-cache.dbm*
 html-docs
 masher/
 Vagrantfile
+.python
+*.idea/*
```

Comment:
This change is OK in general, but let's make it in a separate commit so that this commit remains focused on just renaming the module.